changelog/15277.txt

```diff
@@ -1,11 +1,6 @@
 ```release-note:feature
-**Allows Multiple Issuer Certificates to enable Non-Disruptive
-Intermediate/Root Certificate Rotation**: This introduces /keys and /issuers
-endpoints to allow import, generation and configuration of any number of keys
-or issuers that can be used to issue and revoke certificates.  Keys and Issuers
-can be referred to by (a) a unique UUID; (b) a name; (c) “default”.  If an
-issuer existed prior to this feature, that issuer will be tagged by a migration
-as “default” to allow backwards compatible calls which don’t specify an issuer.
-Creation of new roles will assume an issuer of “default” unless otherwise
-specified.  This default can be configured at /config/issuers and /config/keys.
+**Non-Disruptive Intermediate/Root Certificate Rotation**: This allows
```

Comment:
@sgmiller Reworked the feature changelog entry based on a previous comment of yours in the original [big PR](https://github.com/hashicorp/vault/pull/15277/files#r866301293). Let me know if this is what you had in mind.